src/main/java/com/load/entities/Payload.java

```diff
@@ -0,0 +1,189 @@
+package com.load.entities;
+
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+
+
+import com.fasterxml.uuid.Generators;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+
+@Entity
+@Table(name = "payloads")
+@Data
+public class Payload {
+
+	
+	@Id
+	@GeneratedValue
+	private long id;
```

Comment:
you cannot use @GenerateValue to a string instead you should assign it as "load:" + UUID.randomUUID( ).